### PR TITLE
Fixed a misleading variable name in the doc for enabling the ui

### PR DESCRIPTION
**Pull request by mvisonneau**
Just a documentation change in order to fix a typo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ By default, with no parameters the module will configure vault with some sensibl
 
 * `max_lease_ttl`: A string containing the max lease TTL (default: `undef`)
 
-* `ui`: Enable the vault UI (requires vault 0.10.0+ or Enterprise) (default: `undef`)
+* `enable_ui`: Enable the vault UI (requires vault 0.10.0+ or Enterprise) (default: `undef`)
```
Comment (jsok) [inline README.md]: It is `ui` to match the actual configuration option: https://www.vaultproject.io/docs/configuration/index.html#ui

**Review by jsok — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mvisonneau) [reply README.md]: alright, I updated it to be coherent then 👌

**Comment by ojongerius** [comment]
@jsok good to merge?

**Comment by jsok** [comment]
I completely misunderstood the original PR, asking for the change was not necessary. Apologies @mvisonneau . I've pushed the fix to master just now.

**Comment by mvisonneau** [comment]
no worries, thanks @jsok

**Pull request closed without merging**